USER_GUIDE.md

```diff
@@ -108,6 +109,94 @@ client
     .await?;
 ```
 
+### Make Raw Json Requests
+
+To invoke an API that is not supported by the client, use the `client.send` method to do so. See [examples/json](./opensearch/examples/json.rs) for a complete working example.
+
+#### GET
+The following example returns the server version information via `GET /`.
+```rust
+let info: Value = client
```

Comment:
Not necessarily the issue, may be using Future combinators (`use futures::TryFutureExt;`) would  be a bit more concise as an example:
 
```suggestion
    let info: Value = client.send::<(), ()>(
        Method::Get,
        "/",
        HeaderMap::new(),
        None,
        None,
        None,
    )
    .and_then(|r| r.json::<Value>())
    .await?;
```